Features/Users/users.feature

```diff
@@ -67,3 +67,10 @@ Feature: Use the eZ Users field
         And I fill in "Password" with "12345"
         And I fill in "Confirm password" with "123456"
         Then I should see error messages
+
+    @javascript @edge
+    Scenario: Validate that users have the "Delete" button available
+        Given I am on the Users page
```

Comment:
`When I am on (valid) User page` it is not as generic and it really is what happened. I navigated to the User page so I am on the said user page. However this sentence might cause conflicts with the `Given I am on the Users page` behat does not like similar sentences.